js/react/conversation/Message.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+
+/**
+ * A placeholder Message component, giving the structure of a plain message with none of
+ * the dynamic functionality. We can build off of this going forward.
+ */
+export class Message extends React.Component<{}, {}> {
+  public render() {
+    return (
+      <li className="entry outgoing sent delivered">
+        <span className="avatar" />
+        <div className="bubble">
+          <div className="sender" dir="auto" />
+          <div className="attachments" />
+          <p className="content" dir="auto">
```

Comment:
Curious: What’s our guideline for setting `dir` attribute? The following article suggests setting it where user enter data: https://www.w3.org/International/questions/qa-html-dir Do we set it here because our app is localized and the templates may be filled with either LTR or RTL text? I don’t have a lot of experience with bidi compatibility 😄

Reply:
Pure history. Not really sure what the reasoning is there. I just copied what messages are generating today!